components/hab/install.sh

```diff
@@ -285,13 +285,38 @@ do_install() {
 # Download location for the temporary files
 tmp_dir="${TMPDIR:-/tmp}/hab"
 
+# use stable channel by default
+channel="stable"
+
+# ## CLI Argument Parsing
+
+# Parse command line flags and options.
+while getopts "c:v:" opt; do
+  case $opt in
+    c)
+      channel=$OPTARG
+      ;;
+    v)
+      version=$OPTARG
+      ;;
+    \?)
+      print_help
+      exit_with "Invalid option: -$OPTARG" 1
+      ;;
+  esac
+done
+
+if [ "$channel" = "unstable" ]; then
```

Comment:
I think we can drop this check and rely on a user to set their own `$HAB_DEPOT_URL` if necessary, otherwise we should set this var to something like `export HAB_DEPOT_URL="${HAB_DEPOT_URL:-https://app.acceptance.habitat.sh/v1/depot}"`